src/components/VuePaycard.vue

```diff
@@ -244,9 +246,15 @@ export default {
         : this.defaultPlaceholder
     },
     getCreditCardImage () {
-      return require(`../assets/images/${this.cardType}.png`)
+      const path = require(`../assets/images/${this.cardType}.png`)
+      return path.default || path
     },
     cardType () {
+      if (this.setType.length) {
```

Comment:
I agree this is a good addition to the component and I'm okay with that.
However this implementation doesn't really make sense to me, as we don't have an image called americanexpress or ways to pass in the type.
To me you should pass the type according to the image's name, and also check if that's valid, so should be something like this:

```js
 cardType () {
      const types = [
        'visaelectron',
        'visa',
        'elo',
        'amex',
        'mastercard',
        'discover',
        'unionpay',
        'troy',
        'dinersclub',
        'jcb',
        'laser',
        'dankort',
        'uatp',
        'mir',
        'hipercard',
        'aura',
        'maestro'
      ]
      const sT = this.setType?.toLowerCase()?.replace(/ /g, '')
      if (sT?.length && types.includes(sT)) return sT
      const number = this.valueFields.cardNumber.replace(/\s+/g, '')
```